Monika After Story/game/event-handler.rpy

```diff
@@ -2104,6 +2104,9 @@ label call_next_event:
             if "love" in ret_items:
                 $ mas_ILY()
 
+            if "prompt" in ret_items:
```

Comment:
can you put this after the quit branch? I think quit should have a higher priority.